GameReadyValidator/src/test/java/pl/przemyslawjakubowski/tajgertim/gamereadyvalidator/gamevalidation/GameReadyValidatorQueueListenerTest.java

```diff
@@ -0,0 +1,60 @@
+package pl.przemyslawjakubowski.tajgertim.gamereadyvalidator.gamevalidation;
+
+import org.springframework.context.ApplicationEventPublisher;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pl.jakubowskiprzemyslaw.tajgertim.models.confirmation.Confirmation;
+import pl.jakubowskiprzemyslaw.tajgertim.models.confirmation.FleetPlacementConfirmation;
+import pl.jakubowskiprzemyslaw.tajgertim.models.confirmation.GameConfigurationConfirmation;
+import pl.jakubowskiprzemyslaw.tajgertim.models.confirmation.PlayerConfigurationConfirmation;
+import pl.jakubowskiprzemyslaw.tajgertim.services.LoggerService;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+@Test
+public class GameReadyValidatorQueueListenerTest {
```

Comment:
Apply some formatting.
Try to use Arrange, Act, Assert
Inline redundant variables

Reply:
done